### ISSUE TITLE
Unable to unset metadata layer for Timeseries layer

### ISSUE THREAD

**Issue by jampukka** [oskariorg/oskari-docs#260]
When using the "Date and Range" Time series UI option in Visualization tab for a Timeseries layer the admin user can't properly unselect the metadata layer. The `-------` option doesn't actually clear the selected layer, instead it still gets POSTed as
```
"options": {
  "timeseries": {
    "metadata": {
      "layer": ""
    },
    "ui": "range"
  }
}
```
Changing to Player/animation doesn't clear the `metadata` element from `options` either.

This causes an issue with oskari-server causing it to fail when trying to delete ANY layer from the instance from the following chain:
https://github.com/oskariorg/oskari-server/blob/2.5.1/control-admin/src/main/java/fi/nls/oskari/control/admin/LayerAdminHandler.java#L131
https://github.com/oskariorg/oskari-server/blob/2.5.1/control-admin/src/main/java/fi/nls/oskari/control/admin/LayerAdminHandler.java#L150
https://github.com/oskariorg/oskari-server/blob/2.5.1/control-admin/src/main/java/fi/nls/oskari/control/admin/LayerAdminHelper.java#L28

Current workaround is to temporarily select _any_ layer as the metadata layer and then deleting and re-adding the timeseries WMS layer to remove the invalid metadata element.

**Comment by jampukka** [comment]
I will contribute a fix to `LayerAdminHelper` but proper fix would also include possibility to properly unset the metadata field via UI.

**Comment by ZakarFin** [comment]
Thanks for reporting this!

**Comment by ZakarFin** [comment]
Merged oskariorg/oskari-frontend#1715 for 2.6 release to prevent frontend admin-functionality from generating unnecessary configuration for timeseries layers when admin toggles through the UI.
